### DISCUSSION TITLE
Signup is taking me another website

### DISCUSSION THREAD

**Discussion by umeshEtelligense** [mastodon/mastodon#30234] [General]
Hi Support,

I have installed the Mastodon application on my server, but when I click on "Sign up" from h**ttps://social.icdn.ai/,** it redirects me to the **https://joinmastodon.org/#getting-started** website. Additionally, when I log in with my admin account, it shows me the message: "**Your application is pending review by our staff. This may take some time. You will receive an email if your application is approved.**" Could you please advise on how to resolve this issue?

**Comment by StrangeMinds1960** [comment]
When you say 'Sign up', do you mean 'Create account'?  This has been disabled by admin

For the login, I read somewhere that in the first instance you have to manually approve it via the admin cli
tootctl accounts modify netstairs --approve

https://docs.joinmastodon.org/admin/tootctl/#accounts-modify
